src/khoj/interface/web/login.html

```diff
@@ -0,0 +1,197 @@
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0 maximum-scale=1.0">
+        <title>Khoj - Search</title>
```

Comment:
Should the title say "Khoj - Config"?

```suggestion
        <title>Khoj - Config</title>
```